crates/astria-bridge-withdrawer/tests/blackbox/helpers/mock_bridge_signer.rs

```diff
@@ -0,0 +1,147 @@
+use std::{
+    net::SocketAddr,
+    sync::Arc,
+};
+
+use astria_core::{
+    self,
+    generated::astria::signer::v1::{
+        frost_participant_service_server::{
+            FrostParticipantService,
+            FrostParticipantServiceServer,
+        },
+        GetVerifyingShareRequest,
+        GetVerifyingShareResponse,
+        RoundOneRequest,
+        RoundOneResponse,
+        RoundTwoRequest,
+        RoundTwoResponse,
+    },
+};
+use astria_eyre::eyre::{
+    self,
+    WrapErr as _,
+};
+use frost_ed25519::round1;
+use rand::rngs::OsRng;
```

Comment:
For tests avoid a completely random seed - you can replace this by `rand_chacha` and for example instantiate like ` let rng = rand_chacha::ChaChaRng::seed_from_u64(seed);` via the trait method `rand::SeedableRng::seed_from_u64`. We use it elsewhere in the codebase already.